diff --git a/LKM-Source/rekernel.c b/LKM-Source/rekernel.c
index 76cba36..e9ab2b0 100644
--- a/LKM-Source/rekernel.c
+++ b/LKM-Source/rekernel.c
@@ -20,7 +20,7 @@
 #include "rekernel.h"
 #include <trace/hooks/binder.h>
 #include <trace/hooks/signal.h>
-#include <../../android/binder_internal.h>
+#include <android/binder_internal.h>
 #include <linux/string.h>
 #include <linux/init.h>
 #include <linux/types.h>
